lib/query-interface.js

```diff
@@ -20,13 +20,13 @@ var QueryInterface = function(sequelize) {
 QueryInterface.prototype.createSchema = function(schema, options) {
   options = options || {};
   var sql = this.QueryGenerator.createSchema(schema);
-  return this.sequelize.query(sql, {logging: options.logging});
+  return this.sequelize.query(sql, {logging: options.logging, transaction: options.transaction});
```

Comment:
I actually started doing it that way, but thought there might be some reason the other options besides logging were being stripped. If it's ok to pass along all options, I can update the PR to do that.